lib/request-handler.js

```diff
@@ -29,6 +30,11 @@ const requestCache = new LruCache(1000);
 // Deep error handling for vendor hooks.
 const vendorDomain = domain.create();
 vendorDomain.on('error', err => {
+  Raven.captureException(err, function (sendErr) {
+    if (sendErr) {
+      console.error('Failed to send captured exception to Sentry', sendErr.stack);
+    }
```

Comment:
(I drafted that comment a couple days ago but realized it hadn't posted. Sorry about that!)

Reply:
Would you like to sent errors in batches? I haven't found option to configure when errors are send. We can configure `sampleRate`, `maxReqQueueCount` and `sendTimeout` ([doc](https://docs.sentry.io/clients/node/config/#optional-settings))
Should we add option to configure these options by env variables?

Reply:
No, I think the error reporting -> Sentry here is fine.

What I'm thinking is this: we're inside an if statement for errors that occur while sending errors to Sentry. If Sentry is down and there are a lot of these errors, we don't need a log message for each one.

Reply:
Good point. In this case we will log 2 x messages, since we are already logging `log.error('Vendor hook error:', err.stack);` below. I will think how to throttle these console errors.

Reply:
This can be a follow-up, I feel. It's not a blocker to me.

Reply:
done